### ISSUE TITLE
Unable to add creature data

### ISSUE THREAD

**Issue by cbortz** [pwillworth/galaxyharvester#25]
**Server:** Empire in Flames

When I try to add a creature resource type, the server returns a `500 Internal Error` response.

For example, I was trying to add this resource: http://galaxyharvester.net/resourceType.py/bone_mammal_taanab

**Form Data**
```
resourceType: "bone_mammal_taanab"
galaxy: "64"
creatureName: "Bantha"
harvestYield: "1"
missionLevel: "1"
forceOp: ""
```

![image](https://user-images.githubusercontent.com/184307/127257897-ec128d85-257a-4035-9b5b-76a8cd47f38a.png)

**Comment by pwillworth** [comment]
Looks like a syntax error in the exception handling caused the server error.  I have corrected that here [fix commit](https://github.com/pwillworth/galaxyharvester/commit/2276d5f952bc855e90bf86dfa31e4a94ec8acdca), should work now if you want to try again @cbortz

**Comment by cbortz** [comment]
Yep. Looks to have been resolved by 2276d5f952bc855e90bf86dfa31e4a94ec8acdca. Thanks Paul!
